Sources/System/MachPort.swift

```diff
@@ -0,0 +1,327 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+#if $MoveOnly && (os(macOS) || os(iOS) || os(watchOS) || os(tvOS))
+
+import Darwin.Mach
```

Comment:
Does this need to be `@_implementationOnly`? If so, does that forbid these types from being used in API without either our own compatible type aliases and/or `RawRepresentable` wrappers?

We try hard to keep System as a formal layer above `Darwin` without pulling a bunch of stuff into the global namespace. I don't think we want to cross this bridge here. @lorentey ?

Reply:
I think this is fine! We initially misunderstood the semantics here -- publicly importing Darwin doesn't actually reexport its contents as a side effect. IIRC, at some point before the first release we realized this and switched it all back to regular imports. (Which allows us to refer to the original C types in the public API surface -- which would've been impossible to legally do otherwise.)

For example, the code below will not build unless the `import Darwin.Mach` statement gets uncommented:

```
import SystemPackage
//import Darwin.Mach

func t() {
    let _ = mach_port_name_t() // Cannot fine 'mach_port_name_t' in scope
}
``` 

So all is well!

(Importing `XCTest` or `Foundation` will, however, pollute local name lookup with symbols from `Darwin` -- IIRC, due to an unfortunate wrinkle of clang's module semantics. Oh well.)